jolly-jellyfish/src/page_maker/templates/page_maker/webpage_create.html

```diff
@@ -0,0 +1,14 @@
+{% extends "page_maker/base.html" %}
+
+{% block content %}
+<h2>Create page</h2>
```

Comment:
```suggestion
<h2>Create your page!</h2>
```

Put some energy into it :)